### DISCUSSION TITLE
On blog index and search results templates should we provided autocomplete text fields on the taxonomy drop-down lists

### DISCUSSION THREAD

**Discussion by SophieAmnesty** [amnestywebsite/humanity-theme#77] [Enhancement Requests]
When I am selecting an item on the drop-down list it can take a long time to scroll down to the item I'm looking for. 

Would be great if you could start typing the name of the term to find it more quickly. 

![scrolling down the drop down list](https://user-images.githubusercontent.com/99269799/191750132-d6500133-1740-487b-a46d-a06f3ba4e0f4.gif)

**Comment by jaymcp** [comment]
that's a great idea!

**Comment by AmnestyAM** [comment]
note: not alogia, not on text field
